static/css/base.css

```diff
@@ -75,12 +75,9 @@ body > header {
 	border-radius: 6px;
 	list-style: none;
 	margin: 0;
-	padding: 3px 10px 3px 45px;
+	padding: 6px 10px 6px 45px;
```

Comment:
After the merge of my PR with the now more or less reverted change (#136) I for myself was a bit nerved by the slim padding. I made this change because of the compression of the text lines with a greater font size than [the intended 11px](https://github.com/OpenTTD/website/blob/master/static/css/base.css#L86) (I use a minimal font size of 14px in my browser).

Now it looks more pleasing again but IMHO the site should take dissenting font and element sizes into account in a futury layout.